datasets/streaming/error_aggregates/intro.md

```diff
@@ -0,0 +1,17 @@
+The `error_aggregates` table represents counts of errors counted from main and crash
+pings, aggregated every 5 minutes.
+
+#### Contents
+
+The `error_aggregates` table contains counts of raw counts of various error measures,
+aggregated across each unique set of dimensions (for example: channel, operating system).
+You can get an aggregated count for any particular set of dimensions by summing using SQL.
+
+#### Accessing the data
+
+You can access the data via re:dash. Choose `Athena` and then select the
+`telemetry.error_aggregates` table. Here's an [example query](https://sql.telemetry.mozilla.org/queries/4769/source).
```

Comment:
The chart in the example query is blank.

Reply:
I'll delete the chart. I think that is a good example query in that it shows an example of getting a broad swathe of data, but maybe it would also be useful to do a more targetted query that shows something more graphable as well. I'll add that.